org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/managers/InvisibleProjectImporter.java

```diff
@@ -195,4 +196,12 @@ private IPath inferSourceDirectory(java.nio.file.Path filePath, String packageNa
 
 		return null;
 	}
+
+	/* (non-Javadoc)
+	 * @see org.eclipse.jdt.ls.core.internal.IProjectImporter#getBuildDescriptor()
+	 */
+	@Override
+	public String getBuildDescriptor() {
+		return IProjectDescription.DESCRIPTION_FILE_NAME;
```

Comment:
Is this just a placeholder for the Invisible project?

Reply:
Yes, it is.